src/encoder.rs

```diff
@@ -2095,7 +2094,7 @@ fn encode_tile_group<T: Pixel>(fi: &FrameInvariants<T>, fs: &mut FrameState<T>)
   fs.cdfs = cdfs[idx_max];
   fs.cdfs.reset_counts();
 
-  let max_tile_size_bytes = ((max_len as u32).ilog() + 7) / 8;
+  let max_tile_size_bytes: u32 = ((max_len.ilog() + 7) / 8).try_into().unwrap();
```

Comment:
ilog() usize -> u32 must never fail, you might use the `utils.rs` converter instead.

Reply:
Do you mean we should silently downcast and clamp the value? That seems like it would only create errors further down the line if it ever happened.

Reply:
The maximum value of that expression fits in `u8`.

Reply:
I guess your point is that the `try_into().unwrap()` is inefficient compared to the direct cast. I will change it.